_pages/en/web-services-access.md

```diff
@@ -1,63 +1,57 @@
+The GC Digital Academy and Free Agents, as well as other groups, frequently use Google Forms for registration.
 
-### Background
+In order to improve ESDC's ability to leverage current Web tools, a more open access to Web tools or Software as a Service (SaaS) for IITB employees, as a step toward broader open access to the Internet for more ESDC employees.
 
-The upcoming [Policy on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32603) replaces the [Policy on Acceptable Network and Device Use (PANDU)](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=27122) but keeps the intent to "provide open access to Internet tools and services to support public servants in their work duties, enhance collaboration and networking with their peers".
-The policy applies only to unclassified interactions (data, information, code, etc.).
+As IITB employees generally don't deal with protected information and are well informed and trained on the management of information, the risk would be minimal.
+In addition, the benefits should bring great value such as enhancing collaboration and efficiency, increasing job satisfaction and helping attract and retain IT talent.
+It would also avoid having to use workarounds or alternative sites.
 
-### Issue
+The following Web tools can already be accessed on the ESDC network and are being used by employees (See [Appendix B](#appendix-b---web-tools-already-accessible)).
 
-ESDC is currently blocking access to some popular and useful collaboration Web tools like Google Docs, Google Hangouts and OneDrive.
-Even though TBS policy recommends open access to these tools by default, ESDC's rationale for blocking them is primarily to prevent an accidental leak of protected information by employees.
+### Background
 
-A proposed mitigation item to reduce this risk is to augment the Data Loss Prevention (DLP) capability with a tool that enforces classification of documents and emails.
-However, as the evaluated options to date are costly, implementation is not being considered for the short term.
+The upcoming [Policy on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32603) replaces the [Policy on Acceptable Network and Device Use (PANDU)](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=27122) but keeps the intent to "provide open access to Internet tools and services to support public servants in their work duties, enhance collaboration and networking with their peers".
+See the [Examples of Acceptable Network and Device Use](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32605).
 
-The following Web tools can already be accessed on the ESDC network and are being used by employees (See [Appendix B](#appendix-b---web-tools-already-accessible)).
+### Current Status
 
-### Considerations
+ESDC is currently blocking access to some popular and useful collaboration Web tools and SaaS like Google Docs, Google Hangouts and OneDrive (see [Appendix A](#appendix-a---web-tools-currently-blocked)).
```

Comment:
One thing is missing is the Terms and Conditions acceptance by an employee of ESDC. This is highlighted in this [iService note regarding this exact thing](http://iservice.prv/eng/finance/purchasing/bulletins/bulletin_2017_6.shtml)

Something I sent to a colleague that asked whether she had approval to use Slack. This is what I mentioned (could be mentioned in the document somehow)

- The employee is accountable. So he/she has the approval to use the SaaS. No permission is needed from anyone as long as you are confortable and understand what you are doing, which is collaborating on the internet. This accountability goes away if you need a paid version of a SaaS because someone at ESDC needs to approve expense
-       You use it for professional use only. Not for official ESDC use. This can be gray. What our team is doing is use those free collaboration tools as a workspace. Not to publish official ESDC materials that are meant for public consumption (like the official ESDC Twitter account, or ESDC Website). Think of it as going to Starbucks or a Tim Horton’s to brainstorm with your team. You do not seek permission to go to those place to brainstorm, you just go. But, you don’t talk about sensitive stuff either because you are in a public space. And if you want to buy a coffee, you buy it from your own pocket (not asking ESDC to fund it)
-       You use those free tools for unclassified information only. If you’re unsure what those are, you’ll need to check with ISB using their Information Classification Tool..